docs/devguide/run_time_configuration.rst

```diff
@@ -36,17 +25,49 @@ OpenDDS configuration is concerned with three main areas:
    Each pluggable transport can be configured separately.
    See :ref:`config-transport` for details.
 
-The configuration file for OpenDDS is a human-readable ini-style text file.
+**********************
+Configuration Approach
+**********************
+
+..
+    Sect<7.1>
+
+Most of OpenDDS is configured through a key-value store.
+Keys are strings that are converted to a canonical form before being stored by 1) converting camel case to snake case, 2) capitalizing all alphabetic characers, 3) replacing all non-alphanumeric characters with underscores, and 4) stripping leading, trailing, and duplicate underscores.
+For example, the key ``!SectionInstance__PROPERTY`` is canonicalized into ``SECTION_INSTANCE_PROPERTY`` before being stored.
```

Comment:
It might be helpful to note that the canonical names of each key are in their documentation.